Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -629,6 +632,15 @@ private void LogEntryDeletion(OperationContext context, ShortHash hash, ContentL
             context.TraceDebug($"Deleted entry for hash {hash}. Creation Time: '{entry.CreationTimeUtc}', Last Access Time: '{entry.LastAccessTimeUtc}'");
         }
 
+        /// <todoc />
+        public abstract GetContentHashListResult GetContentHashList(OperationContext context, StrongFingerprint strongFingerprint);
+
+        /// <todoc />

```

Comment:
I think we should change it.

I usually use this is a "TODO" before checking things in.